docs/connection/db_connection/mssql/prerequisites.rst

```diff
@@ -25,6 +25,18 @@ See :ref:`install-spark` installation instruction for more details.
 
 Connecting to MSSQL
 --------------------
+.. attention::
+   In case of NTLM Authentication to MSSQL the similar error might occur:
+
+   .. code-block:: bash
+
+     com.microsoft.sqlserver.jdbc.SQLServerException: The authenticationScheme ntlm is not valid.
+     at com.microsoft.sqlserver.jdbc.AuthenticationScheme.valueOfString(SQLServerDriver.java:153)
+     at com.microsoft.sqlserver.jdbc.SQLServerConnection.connectInternal(SQLServerConnection.java:1221)
+     at com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(SQLServerConnection.java:866)
+     at com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(SQLServerDriver.java:569)
+
+   Consider using proper version of JDBC driver (for instance, com.microsoft.sqlserver_mssql-jdbc-12.8.1.jre8.jar) and add it to the driver's class path.
```

Comment:
That's the default version since [onETL 0.12.0](https://onetl.readthedocs.io/en/stable/changelog/0.12.0.html) (2024-09-03). Since when there is no such problem in the first place, unless user specified old driver version explicitly.
I don't see the point why this note should be added here, it would never appear in documentation of onETL 0.11.x or below.